### ISSUE TITLE
What are the rules for setting the parameters of vit-tiny's decoder？

### ISSUE THREAD

**Issue by zzzzzzyang** [FlyEgle/MAE-pytorch#12]
Thanks for your work! I’m pretraining the vit-tiny for my own dataset, but i can not determine the setting for decoder's parameters(depth/embed_dim,num_heads), just consistent with vit-base/large/huge or choose some smaller value to make a lightweight decoder?

**Comment by zzzzzzyang** [comment]
Due to the limitation of gpus, i can not try many times. Could you give me some suggestions, thanks a lot. :)

**Comment by BruceInWAlDEN** [comment]
May AdamW with lr=2e-5 works. warmup and lr decrease helps。
